safe/gui/tools/shake_grid/shakemap_converter_dialog.py

```diff
@@ -94,6 +95,9 @@ def __init__(self, parent=None, iface=None, dock_widget=None):
         self.main_stacked_widget.setCurrentIndex(1)
         self.update_warning()
 
+        if not setting('developer_mode'):
+            self.smoothing_group_box.hide()
```

Comment:
Maybe its even better to remove it and then call [adjustsize](http://doc.qt.io/qt-4.8/qwidget.html#adjustSize)

Reply:
Will it make me need to check if the radio buttons exist or not in the `accept` method?